airflow/utils/log/file_task_handler.py

```diff
@@ -73,6 +73,19 @@ def _fetch_logs_from_service(url, log_relative_path):
     return response
 
 
+class TriggerLogsPresentationMode(str, Enum):
```

Comment:
Is the `str` part needed? Most of existing enums in Airflow subclass str for backward compatibility reasons (they were actually str in early versions), but new enums can likely do away with it.

Reply:
i think we could do away, but i think strings may make for friendlier interop with the JS components... wdyt?

Reply:
In API we can always call `.value` to get the string value

Reply:
But my opinion is not _too_ strong, especially if we’re looking to potentially move to FastAPI (not sure if this has an effect but the less the possibility is the better)

Reply:
it's a good point, updated